packages/web-components/fast-foundation/package.json

```diff
@@ -15,29 +15,72 @@
+    "test": "npm run test-node:verbose",
+    "test-node": "mocha --reporter min --exit dist/esm/__test__/setup-node.js './dist/esm/**/*.spec.js' -- --experimental-modules",
+    "test-node:verbose": "mocha --reporter spec --exit dist/esm/__test__/setup-node.js './dist/esm/**/*.spec.js' -- --experimental-modules",
+    "test-chrome": "karma start karma.conf.cjs --browsers=ChromeHeadlessOpt --single-run --coverage -- --experimental-modules",
+    "test-chrome:verbose": "karma start karma.conf.cjs --browsers=ChromeHeadlessOpt --single-run --coverage --reporter=mocha -- --experimental-modules",
+    "test-chrome:watch": "karma start karma.conf.cjs --browsers=ChromeHeadlessOpt --coverage --watch-extensions js -- --experimental-modules",
+    "test-chrome:debugger": "karma start karma.conf.cjs --browsers=ChromeDebugging -- --experimental-modules",
+    "test-chrome:verbose:watch": "karma start karma.conf.cjs --browsers=ChromeHeadlessOpt --coverage --watch-extensions js --reporter=mocha -- --experimental-modules",
+    "test-chrome:verbose:debugger": "karma start karma.conf.cjs --browsers=ChromeDebugging --reporter=mocha -- --experimental-modules",
+    "test-firefox": "karma start karma.conf.cjs --browsers=FirefoxHeadless --single-run --coverage -- --experimental-modules",
+    "test-firefox:verbose": "karma start karma.conf.cjs --browsers=FirefoxHeadless --single-run --coverage --reporter=mocha -- --experimental-modules"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@microsoft/eslint-config-fast-dna": "^1.1.1",
-    "circular-dependency-plugin": "^5.2.0",
+    "@microsoft/api-extractor": "^7.7.13",
+    "@types/chai": "^4.2.11",
+    "@types/karma": "^5.0.0",
+    "@types/mocha": "^7.0.2",
+    "@types/webpack-env": "^1.15.2",
+    "chai": "^4.2.0",
+    "ignore-loader": "^0.1.2",
+    "istanbul": "^0.4.5",
+    "istanbul-instrumenter-loader": "^3.0.1",
+    "jsdom": "^16.2.2",
+    "jsdom-global": "3.0.2",
+    "karma": "^5.0.4",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "^2.0.2",
+    "karma-coverage-istanbul-reporter": "^3.0.0",
+    "karma-firefox-launcher": "^1.3.0",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-source-map-support": "^1.4.0",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^4.0.2",
+    "mocha": "^7.1.2",
     "prettier": "2.0.2",
-    "ts-loader": "^6.2.1",
-    "typescript": "^3.7.5"
+    "rollup": "^2.7.6",
+    "rollup-plugin-filesize": "^8.0.2",
+    "rollup-plugin-terser": "^5.3.0",
+    "rollup-plugin-typescript2": "^0.27.0",
+    "source-map": "^0.7.3",
+    "source-map-loader": "^0.2.4",
+    "ts-loader": "^7.0.2",
+    "ts-node": "^8.9.1",
+    "tsconfig-paths": "^3.9.0",
+    "typescript": "^3.7.5",
+    "webpack": "^4.43.0"
   },
   "dependencies": {
     "@microsoft/fast-element": "^0.8.1",
     "@microsoft/fast-web-utilities": "^4.4.5",
-    "tabbable": "^4.0.0"
+    "tabbable": "^4.0.0",
+    "tslib": "^1.13.0"
```

Comment:
Where do we use tslib?

Reply:
In the places where we use decorators, TS will generate helper functions to enable that feature. Without `tslib` it will generate the same helper function into every file, bloating the end result. So, we can take a dependency on `tslib` and it will import the helpers wherever they are needed instead.